lib/net/ssh/transport/session.rb

```diff
@@ -61,12 +61,12 @@ def initialize(host, options={})
       @bind_address = options[:bind_address] || nil
       @options = options
 
-      debug { "establishing connection to #{@host}:#{@port}" }
-
       @socket =
         if (factory = options[:proxy])
+          debug { "establishing connection to proxy" }
```

Comment:
@imoverclocked i appreciate all the comments, but i'm sure we both can spend our times better than arguing on this any further.

I prefer the message to include host and port eg:
```
establishing connection to #{@host}:#{@port} through proxy
```